crates/key-value-redis/src/store.rs

```diff
@@ -25,10 +26,11 @@ impl KeyValueRedis {
 #[async_trait]
 impl StoreManager for KeyValueRedis {
     async fn get(&self, _name: &str) -> Result<Arc<dyn Store>, Error> {
+        let client = Client::open(self.database_url.clone()).map_err(log_error)?;
```

Comment:
This seems to create a `Client` even if `self.connection` already contains a connection?  On every single `StoreManager::get`?

(Oh, this is so extra connections can be created for CAS operations.  I still kinda feel we should be able to reuse the client we created for the original connection, but maybe I'm off the mark there.)

Reply:
Fixed! That was not intended.